src/pages/docs/creators/ray/setup-tutorial.md

```diff
@@ -74,9 +76,9 @@ ray exec golem-cluster.yaml 'ray status'
 
 ```
 
-Congrats, you have started a Ray on the Golem cluster!
-
+The `ray up` succeeds when the head node is running. The remaining 2 nodes should be launching, but don't have to be fully operational.
```

Comment:
I don't see too much value in adding this info... but also fine with having it... just seems like it's too much information in such an introductory article...

Reply:
The value is difference between quickstart and setup tutorial... Most previewers say there should be more content in the setup tutorial...

Reply:
I would leave it as user is expecting a cluster of 3 nodes...